spec/support_specs/logstash_logging_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'rails_helper'
+
+RSpec.describe LogstashLogging do
+  include TestHelpers::CaptureStdoutHelper
+
+  before {
+    @lograge = Struct.new(:enabled, :base_controller_class, :custom_options, :formatter).new
+    @rails_config = Struct.new(:logger, :lograge).new
+    allow(@rails_config).to receive(:lograge).and_return(@lograge)
```

Comment:
Would using `double` be slightly more expressive here? `rails_config = double("config", lograge: lograge)`